### PR TITLE
Clean up rtop integration

### PR DESCRIPTION
**Pull request by chenglou**
Re-enable tests

### CODE REVIEW

**Comment by chenglou** [comment]
@IwanKaramazow you said this has always been failing for you? Anything suspicious you see in `rtopIntegrationTest.sh`?

File: src/rtop_init.ml [hunk 1]
```diff
@@ -12,7 +12,7 @@ print_string
                  / , _/ _// __ |_\\ \\/ /_/ /    /
                 /_/|_/___/_/ |_/___/\\____/_/|_/
 
-  Execute statements/let bindings. Hit <enter> after the semicolon.
+  Execute statements/let bindings. Hit <enter> after the semicolon. Ctrl-c to quit.
```
Comment (anmonteiro) [inline src/rtop_init.ml]: Ctrl+D?

Comment (chenglou) [reply src/rtop_init.ml]: Damn it! Thanks lol

**Comment by IwanKaramazow** [comment]
Pulled your branch & seems to run locally without any problems 🙌

**Pull request merged**